client/src/reducers/categoricalSelection.js

```diff
@@ -20,7 +20,8 @@ const CategoricalSelection = (
   switch (action.type) {
     case "initial data load complete (universe exists)":
     case "set World to current selection":
-    case "reset World to eq Universe": {
+    case "reset World to eq Universe":
+    case "set clip quantiles": {
```

Comment:
Why is categorical selection affected?

Reply:
This is an explicit UI choice, rather than a hard requirement.  I thought we decided that *all* active selection state was cleared on clip, rather than just part of it.    

In theory, we could clear only continuous, and leave both graph/lasso selection and categorical, but that seems quite confusing from a user viewpoint.

Reply:
Ok, after I played with it a little, I understand and agree this is the right decision. Certainly with Undo/Redo this is the best decision.

Reply:
@csweaver It'll be good to keep an eye on this. If users need to change the clipping all the time since the scales are different on different var columns, it may be frustrating to have categories resetting all the time.